news/2025/2025-04-15-the-followpoint-azer-the-tournament-titan.md

```diff
@@ -0,0 +1,305 @@
+
+That match came with a little extra as well! Unfortunately the video is lost media, but as the match ended, for whatever reason I was given host in the multiplayer room. Everyone else had left... except for rrtyui. If you know this player from this time, you know exactly what he was known for, so of course being gifted this situation I did what needed to be done... I picked [The Big Black](https://osu.ppy.sh/beatmapsets/41823) and readied up. I wasn't really expecting anything to happen, but the stream kept going, the casters were confused but optimistic, and the best player in the game readied up on his iconic map in a 1v1 against some random Canadian guy he probably never heard of.
+
+So we played, the chat was really happy to see rrtyui play this map live, the casters were having fun with the situation as well, and eventually the map was cut-off and never reached the end. Let it be known that on this day, I beat rrtyui on The Big Black (he failed the infamous ending before I did, so the map ended... lol).
+
+It was a completely meaningless situation just for fun, but it meant a lot to me at the time and probably was a big reason why I eventually ended up being so involved with competition in the game.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+What would you say is the greatest gift that osu! has given you?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2155578" style="background-image: url('/wiki/shared/avatars/Azer.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/2155578">Azer</a></p>
+
+The best thing osu! has given me is definitely the sense of community! To be able to join here when it was much smaller and make so many close friendships, getting to see it grow into what it is now, the thousands of people I've met that have come and go, for sure. Almost everything in my life somehow links back to this game. Most of my friends on and offline, my career opportunities, etc. It gave me a space to discover a niche talent and do something interesting with my music & gaming interests (both have been a major part of my life since I was 5 years old). It essentially awakened and defined what my passions are, and allowed me to connect with people worldwide who happen to share them too. I really wouldn't trade such a unique experience for anything.
```

Comment:
```suggestion
The best thing osu! has given me is definitely the sense of community! Being able to join here when it was much smaller and make so many close friends, getting to see it grow into what it is now, the thousands of people I've met that have come and go, for sure. Almost everything in my life somehow links back to this game. Most of my friends on and offline, my career opportunities, etc. It gave me a space to discover a niche talent and do something interesting with my music & gaming interests (both have been a major part of my life since I was 5 years old). It essentially awakened and defined what my passions are, and allowed me to connect with people worldwide who happen to share them too. I really wouldn't trade such a unique experience for anything.
```